test/jdk/sun/misc/JarIndex/JarIndexMergeTest.java

```diff
@@ -39,7 +39,7 @@
 import java.util.jar.JarEntry;
 import java.util.jar.JarOutputStream;
 // implementation specific API
-import jdk.internal.util.jar.JarIndex;
+import sun.tools.jar.JarIndex;
 
 public class JarIndexMergeTest {
```

Comment:
Is this the one remaining test in sun/misc/JarIndex? I think it can be moved to test/jdk/java/util/jar. I think we should change the package name in the summary comment too.

Reply:
This is the only test case, and I move it to test/jdk/java/util/jar.
The summary comment is also changed.